internal/credential/model.go

```diff
@@ -25,6 +25,22 @@ func (c Container) JWTString() string {
 	return string(*c.CredentialJWT)
 }
 
+func (c Container) IsValid() bool {
+	return c.ID != "" && (c.HasDataIntegrityCredential() || c.HasJWTCredential())
+}
+
+func (c Container) HasSignedCredential() bool {
+	return c.HasDataIntegrityCredential() || c.HasJWTCredential()
+}
+
+func (c Container) HasDataIntegrityCredential() bool {
+	return c.Credential != nil && c.Credential.Proof != nil
```

Comment:
could it be possible it is false on HasDataIntegrityCredential and false on HasJWTCredential?

that would basically be an invalid cred ye

Reply:
yep yep the `IsValid` on `VerifyCredentialRequest` does exactly this https://github.com/TBD54566975/ssi-service/pull/135/files#diff-8baae75e72d2476225c78bf29869bd02c8d2864a75de717846d02bb4738f4d9eR228